include/EVT/dev/Encoder.hpp

```diff
@@ -27,18 +27,21 @@ class Encoder {
      */
     Encoder(IO::GPIO& a, IO::GPIO& b, uint32_t range, uint32_t initialPosition, bool rollOver);
 
-    /**
-     * Reads and updates the encoder position.
-     */
-    void update();
-
     /**
      * Returns the current absolute position
      *
      * @return the current position of the encoder, between 0 and range, inclusive
      */
     uint64_t getPosition();
 
+    /**
+     * Sets the range and position of the encoder
+     *
+     * @param newRange the new range for the encoder positions
+     * @param newPosition a new position for the encoder (defaults to 0)
+     */
+    void setRangeAndPosition(uint32_t newRange, uint32_t newPosition = 0);
```

Comment:
I think it would make more sense to not have a default value for this, but if you think there's good reason for it, it's probably okay to leave

Reply:
I just figure in most (if not all) circumstances you'd want the position to reset to 0, but I didn't want to lose the ability to specify if necessary.